cmd/mdatagen/test_metadata.yaml

```diff
@@ -0,0 +1,56 @@
+# Sample metric metadata file with all available configurations.
```

Comment:
Should we just call it `metadata.yaml`? It might be nice because it sort of makes the package demonstrate its own usage. We already have the `internal/metadata/*` files named normally too.

Reply:
Ok, makes sense